docs/user-guide/bifrost/bifrost-reduction.ipynb

```diff
@@ -0,0 +1,73 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# BIFROST reduction of simulated data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import scipp as sc\n",
+    "\n",
+    "from ess.bifrost.data import (\n",
+    "    # simulated_elastic_incoherent_with_phonon,\n",
+    "    tof_lookup_table_simulation,\n",
+    ")\n",
+    "# from ess.spectroscopy.indirect.workflow import bifrost"
```

Comment:
Isn't the name `bifrost` going to potentially cause clashes if someone also does `from ess import bifrost`?